src/pages/Teams/AddUserToTeam/AddUserToTeam.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  Typography,
+  OutlinedInput,
+  Box,
+  Chip,
+  Button,
+} from '@mui/material';
+import GroupAddIcon from '@mui/icons-material/GroupAdd';
+import { getUsers } from '../../../store/actions/users';
+import { getTeams, addUserToTeam } from '../../../store/actions/teams';
+
+const AddUserToTeam = () => {
+  const { teamID } = useParams();
+  const dispatch = useDispatch();
+
+  const { users } = useSelector((state) => state.users);
+  const { teams } = useSelector((state) => state.teams);
+
+  const [selectedUsers, setSelectedUsers] = useState([]);
+  const [selectedTeam, setSelectedTeam] = useState(null);
+  const [selectableUsers, setSelectableUsers] = useState([]);
+
+  useEffect(() => {
+    dispatch(getUsers());
+  }, []);
+
+  useEffect(() => {
+    if (teams.length) {
+      const tempUsers = JSON.parse(JSON.stringify(users));
+      const team = teams.find((t) => t._id === teamID);
+
+      const alreadySelectedUsers = team.users.map((u) => u.user._id);
+
+      const filteredUsers = tempUsers.filter(
+        (user) => !alreadySelectedUsers.includes(user._id)
+      );
+
+      setSelectableUsers(filteredUsers);
+      setSelectedTeam(team);
+    } else {
+      dispatch(getTeams());
+    }
+  }, [teams, users]);
+
+  const handleChange = (event) => {
+    setSelectedUsers(event.target.value);
+  };
+
+  const getStyles = (name) => ({
+    fontWeight: selectedUsers.indexOf(name) === -1 ? 500 : 800,
+  });
+
+  const addUsers = async () => {
+    await selectedUsers.forEach((user) => {
+      dispatch(addUserToTeam(teamID, user._id));
+    });
+
+    setSelectedUsers([]);
+  };
+
+  const capitalize = (text) => text.charAt(0).toUpperCase() + text.slice(1);
+
+  return (
+    <div className="add-user-to-team">
+      <Typography variant="h5" component="h5">
+        Add User To Team {selectedTeam && selectedTeam.name}
+      </Typography>
+      <FormControl
+        fullWidth
+        sx={{
+          margin: '20px 0',
+        }}
+      >
+        <InputLabel id="demo-simple-select-label">Available Users</InputLabel>
```

Comment:
Instead of InputLabel add label props in select to avoid styling problem :)

Reply:
<img width="166" alt="Capture d’écran 2022-12-21 à 11 07 58" src="https://user-images.githubusercontent.com/48067485/208879077-b9e536bf-ecca-4d2b-8bc7-8bec7c1c6cc4.png">

This is not working apparently. I will reduce the label length to 'Users'